README.md

```diff
@@ -54,4 +57,4 @@ Reviewers:
 1. Sidharth Babu
 2. Roie Gal
 3. Ishan Deshpande
-4. Nathaniel Delgado
```

Comment:
I see how it is

Reply:
That's what I get for not pulling from remote :/